modules/gospel.js

```diff
@@ -252,6 +252,7 @@ const Module = new Augur.Module()
 .addEvent("messageCreate", msg => {
   if (!msg.inGuild()) return;
   if (msg.channel.parent?.id === u.sf.channels.gospelCategory && !u.parse(msg) && !msg.author.bot) {
+    searchExp = new RegExp(`\\b(${[...abbreviationTable.keys()].join("|")})\\W(\\d+)\\W?([\\d\\-;,\\W]+)`, "ig");
```

Comment:
Just define it here and get rid of the definition in setInit (229) and at line 25
```suggestion
    const searchExp = new RegExp(`\\b(${[...abbreviationTable.keys()].join("|")})\\W(\\d+)\\W?([\\d\\-;,\\W]+)`, "ig");
```